src/main/scala/uk/gov/hmrc/play/bootstrap/FrontendModule.scala

```diff
@@ -31,7 +31,7 @@ class FrontendModule extends BootstrapModule {
     super.bindings(environment, configuration) ++ Seq(
       bind[AuditFilter].to[FrontendAuditFilter],
       bind[SessionCookieCrypto].toProvider[SessionCookieCryptoProvider],
-      bind[ApplicationCrypto].to[ApplicationCryptoDI],
+      bind[ApplicationCrypto].to(new ApplicationCrypto(configuration.underlying)),
```

Comment:
you could change `to` to `toInstance` to only create a single instance of `ApplicationCrypto`